pkg/spoke/submarineragent/config_controller_test.go

```diff
@@ -42,6 +44,14 @@ var _ = Describe("Config Controller", func() {
 	testManagedClusterAddOn(t)
 })
 
+func labelGateway(node *corev1.Node, isGateway bool) {
+	if isGateway == true {
```

Comment:
This is odd, I think one of the other linters should have flagged it — it should be `if isGateway {`. But see my suggestion...